### ISSUE TITLE
[LC0003] False positive 

### ISSUE THREAD

**Issue by fvet** [StefanMaron/BusinessCentral.LinterCop#165]
![image](https://user-images.githubusercontent.com/25268332/155893188-e9830281-cdf8-4a0b-b663-ba65427cdb49.png)

Warning(s)

- _Do not use an Object ID for properties or variables declaration. Use Option instead._
- _The message for the diagnostic '0' could not be formatted. Message: 'There was an Error in Rule "{0}" of type "{1}" {2}'. Arguments: 'Rule0003, unknown'._

Code

```
[EventSubscriber(ObjectType::Codeunit, Codeunit::ReportManagement, 'OnAfterDocumentPrintReady', '', false, false)]
    local procedure OnAfterDocumentPrintReady(ObjectType: Option; ObjectID: Integer; ObjectPayload: JsonObject; DocumentStream: InStream; var Success: Boolean);
```

**Comment by fvet** [comment]
Similar warning on following local var definition

`State: Option Operator,Number;`

**Comment by StefanMaron** [comment]
I do not get any message when I use your example code. 
But I had an idea what causes the LC0000 to not print correctly.
Please make sure you have version https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.24.9 installed and report back if the LC0000 shows up correctly.

**Comment by fvet** [comment]
Hi @StefanMaron , I've attached a sample repo with minimum code / settings that results in the above reported warning, which seems still present in pre-release v0.24.9. If you need more info, glad to help.

![image](https://user-images.githubusercontent.com/25268332/159169261-8a54c24d-63f1-4eed-ba0d-ab532d46588f.png)

[ALProject8.zip](https://github.com/StefanMaron/BusinessCentral.LinterCop/files/8311520/ALProject8.zip)

**Comment by StefanMaron** [comment]
I still dont get any warning
![image](https://user-images.githubusercontent.com/24838311/159175518-4816829b-9eb6-4248-aa10-45f4c849c792.png)

Do you have the same AL Language Verson?
![image](https://user-images.githubusercontent.com/24838311/159175547-5c0321a7-26e5-44ae-a02e-b1d3352e6fcc.png)

**Comment by fvet** [comment]
Forgot to mention, I'm running one of the BC insider 9.x versions.

_v9.0.592718_
